docker-images/kafka/Dockerfile

```diff
@@ -39,6 +39,9 @@ COPY kafka-thirdparty-libs/target/dependency/ ${KAFKA_HOME}/libs/
 
 RUN yum -y install stunnel net-tools && yum clean all -y
 
+# for nslookup
+RUN yum -y install bind-utils && yum clean all -y
```

Comment:
Could you just add the package to the line 40 so that all the installations are done in a single command?

Reply:
My bad! I should have added it to the command on line 40. Thank you for the comment :)